xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -452,17 +454,46 @@ bool CGUIWindowPVRGuideBase::OnMessage(CGUIMessage& message)
   return bReturn || CGUIWindowPVRBase::OnMessage(message);
 }
 
+bool CGUIWindowPVRGuideBase::OnActionContextMenu()
```

Comment:
We should rename the method or move the code to the case `ACTION_CONTEXT_MENU` because it's only used for gap items. At the moment, it's a little confusing because I thought it handles context menu actions for all items.

Reply:
Do you have a suggestion for a better name?